### DISCUSSION TITLE
Over shooting set temperature

### DISCUSSION THREAD

**Discussion by crazynightgriffin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#468] [Q&A]
Have got the system setup but there seems to be a delay in turning the heat off, from some searching around I have read that there is a 2'c +/- range before the heater will turn on or off.

Is there a simple work around for this as the less the heater is on the more money I will save :)

I do have HA and node red so any automations/scripts people can share would be really helpful.

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/66025008/225138125-03e72aac-c822-4ebd-9643-8cde869d2e21.png">

**Comment by visualapproach** [comment]
I haven't tried but you should be able to make an automation that turns off heat when virtual temp is more than say 38.1 and another automation that turns it on again below 37.9. Probably works best if you max the target temp on the pump (40) so you don't have two systems fighting each other.

**Reply by crazynightgriffin** [reply]
Will have a play in a few days time, at the moment happy it's working,  I was thinking could just trigger the layzspa_heat_regulation switch off once the temperature is hit and back on when it drops below.
A whole new climate device targeting the switch and sensors might work
<img width="499" alt="image" src="https://user-images.githubusercontent.com/66025008/225151231-6ee34dc2-90d3-48f9-b144-59ecb745deb7.png">
